### ISSUE TITLE
Is there a way of showing ONLY the results that have matched?

### ISSUE THREAD

**Issue by ebarronh** [EddyBorja/MLPAutoCompleteTextField#68]
I already have a very long list of numbers and I'd like to see if there is anyway I can only show the results that have matched (at least by 1 letter) what the user has entered?
Any suggestions?

**Comment by tommybananas** [comment]
+1

**Comment by Hummer101** [comment]
+1

**Comment by StoryStar** [comment]
+1

**Comment by pyraego** [comment]
+1

**Comment by NirmitDagly** [comment]
+1

**Comment by EddyBorja** [comment]
Can you give an example of a user input and an expected list of results?



—

**Comment by NirmitDagly** [comment]
Yes.
For example, Suppose I am searching a word "Love" in an array, and If I enter the text "Lo" then it should show results from starting of this words and not containing this words.

**Comment by EddyBorja** [comment]
So basically looking for an exact match and not considering typos?



—

**Comment by NirmitDagly** [comment]
Yes.. Exactly. Looking for an exact match rather then finding out typos from an array.

**Comment by EddyBorja** [comment]
I've pushed an update that adds a Bool called "requireAutoCompleteSuggestionsToMatchInputExactly".
Set that to true and it should do the behavior you're looking for. 
An exact match is the same as having an edit distance of zero.

**Comment by NirmitDagly** [comment]
Thanks for your commit Eddy. Now, I am getting exact words that are matches with the latter that I want. But It is not finding proper words from my data source. I have an array of total 16042 objects and I want to find out the search words. But somehow it is just searching words till the latter starts from "B' and then after other words are being ignored. I have used NSPredicate and also used string by based on its range to narrow my search for the words, but it is also not working. I can provide data source also if you need it. Please help me out to solve this.

Once again thanks for your help.

**Comment by EddyBorja** [comment]
Sure let me get a data source so I can see what's going on.



—

**Comment by hardikamal** [comment]
hi @EddyBorja,
Can u plz tell how do i make it match case-insensitive words... suppose my word is "Apple" and if i type "a",the Apple should appear...Thanks

**Comment by hardikamal** [comment]
@EddyBorja  any solution plz..
